starfish/spots/_detector/detect.py

```diff
@@ -144,7 +142,7 @@ def concatenate_spot_attributes_to_intensities(
     n_round: int = max(inds[Indices.ROUND] for _, inds in spot_attributes) + 1
 
     all_spots = pd.concat([sa.data for sa, inds in spot_attributes])
-    spot_attribute_index = dataframe_to_multiindex(all_spots.drop(['spot_id', 'intensity'], axis=1))
+    features_coordinates = all_spots.drop(['spot_id', 'intensity'], axis=1)
```

Comment:
i had to stare at this awhile. maybe add a comment, that this only keeps columns: x, y, z, radius, quality? or something like that.